src/store/authors/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import getHeaders from '../../api';
+import BASE_URL from '../../env';
```

Comment:
This is completely wrong. Learn how to work with environment variables in React
https://create-react-app.dev/docs/adding-custom-environment-variables/
https://dev.to/felixdusengimana/environment-variables-in-reactjs-139l